docs/man-openmpi/man3/MPI_Status_get_error.3.rst

```diff
@@ -52,16 +52,31 @@ INPUT PARAMETER
 
 OUTPUT PARAMETER
 ----------------
-* ``error``: error set in the MPI_ERROR field (integer).
+* ``error``: error set in the ``MPI_ERROR`` field (integer).
 * ``ierror``: Fortran only: Error status (integer).
 
 DESCRIPTION
 -----------
 
-Returns in error the MPI_ERROR field from the status object.
+Returns in error the ``MPI_ERROR`` field from the ``status`` object.
+
+While the ``status`` object members ``MPI_SOURCE``, ``MPI_TAG``, and
+``MPI_ERROR`` are directly accessible in C and Fortran, for
+convenience in other contexts (e.g., when using alternate MPI bindings
+in languages that do not directly translate the ``status`` object),
+users can also access these values via procedure calls such as this
+one.
 
 
 ERRORS
 ------
 
 .. include:: ./ERRORS.rst
+
+
+.. seealso::
+   * :ref:`MPI_Status_get_source`
+   * :ref:`MPI_Status_get_tag`
+   * :ref:`MPI_Status_set_error`
+   * :ref:`MPI_Status_set_source`
+   * :ref:`MPI_Status_set_tag`
```

Comment:
If we decide to add the related functions, we should probably also list `MPI_Status_set_cancelled` which was defined already in an earlier version of the MPI spec. And not sure about `MPI_Test_cancelled` (which is kind of `MPI_Status_get_cancelled` but with the old nomenclature), as well as `MPI_Get_count` and `MPI_Get_elements`. I couldn't find a function to set the value for count, but I also didn't do an exhaustive search.

Reply:
`MPI_Status_set_elements` and `MPI_Status_set_elements_x` ?